Code/Editor/Include/IFileUtil.h

```diff
@@ -60,7 +60,6 @@ struct IFileUtil
         EFILE_TYPE_GEOMETRY,
         EFILE_TYPE_TEXTURE,
         EFILE_TYPE_SOUND,
-        EFILE_TYPE_GEOMCACHE,
```

Comment:
It looks like this `ECustomFileType` enum is actually unused now (the `SelectSingleFile` methods are commented in the code), so maybe we could just remove the whole thing?

Reply:
Yeah, I just didn't want to get sidetracked pulling out all the unused code. I have a few more critical tickets to get to right now.